.github/workflows/linux.yml

```diff
@@ -360,7 +381,58 @@ jobs:
             ${{ env.OPENVINO_REPO }}/TEST*.html
             ${{ env.OPENVINO_REPO }}/TEST*.xml
           if-no-files-found: 'warn'
-          
+
+
+  store_artifacts:
+    name: Store build artifacts
+    needs: [openvino_download, openvino_tokenizers_wheel, openvino_tokenizers_cpack]
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-2-cores-8gb
+    container:
+      image: openvinogithubactions.azurecr.io/library/python:3.12-slim
+      volumes:
+        - /mount:/mount
+        - ${{ github.workspace }}:${{ github.workspace }}
+    env:
+      CPACK_PACKAGE: ${{ github.workspace }}/ov_tokenizers.tar.gz
+      WHEEL_PACKAGE: ${{ github.workspace }}/wheels
+      MANIFEST_PATH: ${{ github.workspace }}/manifest.yml
+
+    steps:
+      - name: Download tokenizers package
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: openvino_tokenizers_cpack_Release
+          path: ${{ github.workspace }}
+
+      - name: Download tokenizers wheels
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: openvino_tokenizers_wheel
+          path: ${{ env.WHEEL_PACKAGE }}
+
+      - name: Download manifest
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          name: manifest
+          path: ${{ github.workspace }}
```

Comment:
I would suggest to merge those steps in one, like:

      - name: Download OpenVINO artifacts
        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
        with:
          pattern: "{openvino_package,openvino_wheels,openvino_developer_package,openvino_js_package}"
          path: ${{ env.ARTIFACTS_DIR }}
          merge-multiple: true

Reply:
Done for 2 out of 3 artifacts that're downloaded to the same place (I need still need wheels in a dedicated folder)